### ISSUE TITLE
Error: request entity too large

### ISSUE THREAD

**Issue by bwl21** [voidlabs/mosaico#35]
When I run mosaico against the local webserver and create a big newsletter, then I get this. Is it an issue of the backend?

Error: request entity too large
   at readStream (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/node_modules/raw-body/index.js:196:17)
   at getRawBody (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/node_modules/raw-body/index.js:106:12)
   at read (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/lib/read.js:68:3)
   at urlencodedParser (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/lib/types/urlencoded.js:109:5)
   at Layer.handle [as handle_request] (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/express/lib/router/layer.js:95:5)
   at trim_prefix (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/express/lib/router/index.js:312:13)
   at /Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/express/lib/router/index.js:280:7
   at Function.process_params (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/express/lib/router/index.js:330:12)
   at next (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/express/lib/router/index.js:271:10)
   at jsonParser (/Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/lib/types/json.js:107:37)

**Comment by bago** [comment]
How big is big? If I create a newsletter including 20 blocks from versafix I can save it against the sample node.js backend without issues.

Have a look here: http://stackoverflow.com/questions/19917401/node-js-express-request-entity-too-large

BTW: if your newsletter HTML is more than 1 MB then you have an issue.. spam filters will not be happy to receive 1 MB of content.. so maybe something else bad is happing?

Also: you didn't say what option gave you this issue (send test? download? Both?)

**Comment by bwl21** [comment]
The issue occurs, when I push "download" in Mosaico. I managed to get it out by removing the check for limit in /Volumes/Macintosh HD/Users/beweiche/beweiche_noTimeMachine/204_ruby-hacks/mosaico/node_modules/body-parser/node_modules/raw-body/index.js line 195 and 285. 

My newsletter is 200k in size. The limit in these two lines come somehow from a configuration, but I could not find out, where to change this. So I did a brute force attack ;-)
